README.md

```diff
@@ -73,49 +147,19 @@ ARGS:
     <paths>...    Sets the input paths to use
 ```
 
-Let's see a few examples, assuming the source directory is `~/Documents/mozilla-central` and the build directory is `~/Documents/mozilla-central/build`.
-
-### LCOV output
-
-```sh
-grcov ~/Documents/mozilla-central/build -t lcov > lcov.info
-```
-
-As the LCOV output is compatible with `lcov`, `genhtml` can be used to generate a HTML summary of the code coverage:
-```sh
-genhtml -o report/ --show-details --highlight --ignore-errors source --legend lcov.info
-```
+## Alternative reports
 
-### Coveralls/Codecov output
+grcov provides the following output types:
 
-```sh
-grcov ~/Documents/FD/mozilla-central/build -t coveralls -s ~/Documents/FD/mozilla-central --token YOUR_COVERALLS_TOKEN > coveralls.json
-```
-
-### grcov with Travis
-
-Here is an example of .travis.yml file
-```YAML
-language: rust
-
-before_install:
-  - curl -L https://github.com/mozilla/grcov/releases/latest/download/grcov-linux-x86_64.tar.bz2 | tar jxf -
-
-matrix:
-  include:
-    - os: linux
-      rust: nightly
-
-script:
-    - export CARGO_INCREMENTAL=0
-    - export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zno-landing-pads"
-    - cargo build --verbose $CARGO_OPTIONS
-    - cargo test --verbose $CARGO_OPTIONS
-    - |
-      zip -0 ccov.zip `find . \( -name "YOUR_PROJECT_NAME*.gc*" \) -print`;
-      ./grcov ccov.zip -s . -t lcov --llvm --branch --ignore-not-existing --ignore "/*" -o lcov.info;
-      bash <(curl -s https://codecov.io/bash) -f lcov.info;
-```
+| Output Type `-t` | Description |
+| ---            | ---         |
+| lcov (default) | lcov's INFO format that is compatible with the linux coverage project. |
+| ade            | ActiveData\-ETL format. Only useful for mozilla projects               |
+| coveralls      | Upload coverage output directly to coveralls.io (requires token parameter). | 
```

Comment:
The description is a bit confusing, this doesn't upload coverage, this is the Coveralls format

Reply:
Also, token is not really mandatory since 989a84bb29e95e392589c4e73c29189fd69a1d4e.

Reply:
Sorry I've never used coveralls - does the updated look better?